internal/backend/remote-state/pg/backend_test.go

```diff
@@ -18,6 +19,8 @@ import (
 	"github.com/lib/pq"
 )
 
+var databaseName string
```

Comment:
Instead of this floating variable could we have the `testACC` function return `u.Path[1:]` and the `TestBackendConfig` test could hold the variable locally?

Reply:
Yes that's much better. I must have thought that I was in an init function not an helper.